client/webserver/site/src/js/wallets.js

```diff
@@ -304,6 +310,23 @@ export default class WalletsPage extends BasePage {
     this.animation = this.showBox(box)
   }
 
+  /* Fetch a new address from the wallet. */
+  async newDepositAddress () {
+    const page = this.page
+    Doc.hide(page.depositErr)
+    app.loading(page.deposit)
+    const res = await postJSON('/api/depositaddress', {
+      assetID: this.depositAsset
+    })
+    app.loaded()
+    if (!app.checkResponse(res, true)) {
+      page.depositErr.textContent = res.msg
+      Doc.show(page.depositErr)
+      return
+    }
+    page.depositAddress.textContent = res.address
+  }
```

Comment:
Appears to work but when clicking the New Deposit Address button the following error appears in the console:

![image](https://user-images.githubusercontent.com/9373513/96923730-51256100-1477-11eb-9f3c-f178cde1b336.png)

Reply:
I've seen this on master. Seems to be benign, but I'll chase it down in another PR, if that's alright.

Reply:
Nevermind. I picked it up.